cfme/tests/containers/test_containers_properties.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# the test performs field validation in Properties table
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.pod import Pod, list_tbl as list_tbl_pod
+from cfme.containers.route import Route, list_tbl as list_tbl_route
+from cfme.containers.project import Project, list_tbl as list_tbl_proj
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+# Polarion test 9911
+# container pods Properties table field verification
+
+
+@pytest.mark.parametrize('rel',
+                         ['name',
+                          'phase',
+                          'creation_timestamp',
+                          'resource_version',
+                          'restart_policy',
+                          'dns_policy',
+                          'ip_address'
+                          ])
+def test_pods_properties_rel(provider, rel):
+    sel.force_navigate('containers_pods')
+    ui_pods = [r.name.text for r in list_tbl_pod.rows()]
+    mgmt_objs = provider.mgmt.list_container_group()  # run only if table is not empty
+
+    if ui_pods:
+        # verify that mgmt pods exist in ui listed pods
+        assert set(ui_pods).issubset(
+            [obj.name for obj in mgmt_objs]), 'Missing objects'
+
+    for name in ui_pods:
+        obj = Pod(name, provider)
+        field_content = getattr(obj.summary.properties, rel).text_value
```

Comment:
Why not to check all the fileds per one page navigation?

    pod = Pod(name, provider)
    assert pod.summary.properties.name.text_value
    assert pod.summary.properties.phase.text_value
    assert pod.summary.properties.creation_timestamp.text_value
    ...

The benefit will be that the tests will finish much faster!